uavcan/pnp/32741.NodeIDAllocationData.0.1.uavcan

```diff
@@ -180,18 +182,18 @@
 # If the message transfer is anonymous (i.e., allocation request), this is the preferred ID.
 # If the message transfer is non-anonymous (i.e., allocation response), this is the allocated ID.
 #
-# If the allocatee does not have any preference, it should request the highest possible node-ID, which is 125,
-# because 126 and 127 are reserved for network maintenance tools. Requesting 126 and 127 is not prohibited,
+# If the allocatee does not have any preference, it should request the highest possible node-ID. Keep in mind that
+# the two highest node ID values are reserved for network maintenance tools; requesting those is not prohibited,
 # but the allocator is recommended to avoid granting these node-ID, using nearest available lower value instead.
 # The allocator will traverse the allocation table starting from the preferred node-ID upward,
 # until a free node-ID is found (or the first ID reserved for network maintenance tools is reached).
 # If a free node-ID could not be found, the allocator will restart the search from the preferred node-ID
 # downward, until a free node-ID is found. In pseudocode:
 #
-#   uint8_t findFreeNodeID(const uint8_t preferred)     // Returns >127 on failure (valid node-ID range from 0 to 127)
+#   uint16_t findFreeNodeID(const uint16_t preferred)   // Returns 0xFFFF on failure
```

Comment:
would a node ID not be an IPv6 address for that transport? I.e. does this need to handle 128 bit addresses?

Reply:
Probably not, but this is very debatable. See https://forum.uavcan.org/t/alternative-transport-protocols/324, search for "determinism considerations".